tests/test_launcher.py

```diff
@@ -75,6 +76,66 @@ def test_container_launcher():
         assert session.health_check_service.is_serving
 
 
+@pytest.mark.standalone
+def test_case_load():
+    # Test that launch_fluent() works with a case file as an argument
+    _, cas_path = download_input_file(
+        "pyfluent/mixing_elbow",
+        "mixing_elbow.cas.h5",
+    )
+    session = pyfluent.launch_fluent(case_file_name=cas_path)
```

Comment:
As @ansjsia identified, the `standalone` marker is required as these tests won't work in the GitHub workflows, because we don't currently support `case_file_name` for container runs, and we don't have tests with a standalone Fluent install. See https://github.com/ansys/pyfluent/issues/2314.